Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -5203,11 +5203,11 @@ size_t Document::broadcast_active_resize_observations()
                 shallowest_target_depth = target_depth;
         }
 
-        // 4. Invoke observer.[[callback]] with entries.
-        observer->invoke_callback(entries);
-
-        // 5. Clear observer.[[activeTargets]].
+        // 4. Clear observer.[[activeTargets]].
         observer->active_targets().clear();
+
+        // 5. Invoke observer.[[callback]] with entries.
```

Comment:
The lhs is the order in the spec: https://drafts.csswg.org/resize-observer-1/#broadcast-resize-notifications-h

If this no longer matches the spec, it probably should have at least a comment on why, and if this is the correct change, a spec bug too?

Reply:
The crash alone doesn't necessarily indicate there is a spec bug, especially with Ladybird being in a pre-alpha state.
I would probably investigate further and gain a more solid foundation / justification for why it needs to be this way (for us and the CSS wg)